sockeye/attention.py

```diff
@@ -336,6 +344,117 @@ def attend(att_input: AttentionInput, att_state: AttentionState) -> AttentionSta
         return attend
 
 
+class MultiHeadDotAttention(Attention):
+    """
+    Dot product attention with multiple heads as proposed in Vaswani et al, Attention is all you need.
+    Can be used with a RecurrentDecoder.
+
+    :param input_previous_word: Feed the previous target embedding into the attention mechanism.
+    :param depth: Number of hidden units.
```

Comment:
ok I can change it here, but depth is used in the transformer code so I will leave it there